### DISCUSSION TITLE
video tutorials source code

### DISCUSSION THREAD

**Discussion by tomschulze** [microstream-one/microstream#385] [Q&A] [answered]
dear microstream team,

thank you for providing the [video tutorials](https://www.youtube.com/playlist?list=PLXEK1LzqVu9jH8iYuuz5zEBCSWd9ZeP37) w/ best practice examples on how to get started. i did learn a lot from them and i think they are a very valuable resource. however, i am missing the source code. i know there exist [code examples](https://github.com/microstream-one/microstream/tree/master/examples) and there's the bookstore demo. unfortunately, none of these seem to contain the best practice code from the video series (data access layer, DB class, etc.).

would you want to share the code? i am particularly interested in the source code from videos 3 to 5.
by the way: you could also provide a link to the videos in the docs. i found them more or less by luck when checking out your you tube channel.

thank you and kind regards,
tom

**Comment by fh-ms** [comment, marked as answer]
The trainer has repositories on his [GitHub](https://github.com/ZorgCK?tab=repositories&q=MSTraining&type=&language=&sort=) which he used in the videos.

**Reply by tomschulze** [reply]
oh. i didn't see that. thanks!

**Comment by guykatz** [comment]
this is 2 years old. are the vids still relevant?
thanks

**Reply by fh-ms** [reply]
The version of MicroStream used in this videos is an older one of course, but the principles and API shown still apply.
